grid2op/tests/test_ma.py

```diff
@@ -273,7 +283,8 @@ def check_n_objects(self, ma_env, domain, space = 'action', add_msg = ""):
                 ma_env._subgrids_cls[space][agent].n_gen+\
                 ma_env._subgrids_cls[space][agent].n_load+\
                 ma_env._subgrids_cls[space][agent].n_line*2+\
-                ma_env._subgrids_cls[space][agent].n_interco, add_msg
+                ma_env._subgrids_cls[space][agent].n_interco+\
+                ma_env._subgrids_cls[space][agent].n_storage, add_msg
```

Comment:
see hopefully we did the test with some storage !

Reply:
Where comments ?

Reply:
:)